### DISCUSSION TITLE
Number of players and more

### DISCUSSION THREAD

**Discussion by vanja-san** [seDirector/App#71] [Ideas]
Can you please add an option to disable overriding of this parameter (for remove `maxplayers_override`), for example, add a checkbox next to it
Example:
![Group 3](https://github.com/seDirector/App/assets/7201687/204789fd-8d2a-4fb9-b672-4bcf9efc734f)

When the window is minimized, the buttons are overlaid on top of the servers, and the top right text (Compare Free vs Pro) is overlaid on top of the menus (need fixed minimum window size)
Example:
![Снимок экрана 2023-07-31 025816](https://github.com/seDirector/App/assets/7201687/202baa88-db9e-4023-957c-756c46b5d17a)

It would be nice to be able to disable columns (mouse right click)
Example:
![Group 9](https://github.com/seDirector/App/assets/7201687/7a8b187e-1a2f-473d-9bbc-c29d4a1e5ad4)

Also the ability to hide the top menu with icons. Or turn off the text and leave only the icons., thereby reducing the size of the menu itself.
Example (Any size of icons):
![Group 11](https://github.com/seDirector/App/assets/7201687/808b0081-f2b8-4e48-919c-eaf0d7cf8b18)

`Sorry for bad english!`

**Comment by DAYBR3AK1999** [comment]
After having a little talk with @vanja-san i understood what he ment:

It would be better if the "Number of Players" value could have an empty value instead of 0 or when chosen 0 no command will be written about "maxplayers_override".

If empty value or value 0 chosen the line "maxplayers_override" should not pop/be added in the "command line to run".

it could possibly fix and optimise the startup code for CSGO Servers and maybe other servers as well.

**Comment by asheroto** [comment]
Got it. I will put this on the roadmap. Thanks.

does this feature need to be available to any other game? or is it specific to CSGO?

**Reply by vanja-san** [reply]
Well, if there's no problem, can do it for everyone. 🤔

**Reply by asheroto** [reply]
yeah I just wasn't sure if it was a needed option for other games?

**Reply by vanja-san** [reply]
I'm not sure either, since I only use csgo. 😅

**Reply by vanja-san** [reply]
Maybe make 24 the default value for everyone, and those who know will choose an empty value.

**Reply by asheroto** [reply]
Will it let us start a game without specify the number of players?

**Reply by DAYBR3AK1999** [reply]
Technically if an empty value is selected it should use the default maxplayers value of the game you created a server for.

So 0 or Empty Space should not write any +maxplayers line unless value 1 is selected.

This might help:

https://developer.valvesoftware.com/wiki/Changing_Maximum_Player_Count

&

https://developer.valvesoftware.com/wiki/Maxplayers

**Reply by mrc4tt** [reply]
@vanja-san if you only host the csgo server, SED will only use `-maxplayers_override` as you can't **set** maxplayers command.

**Comment by asheroto** [comment]
Sorry for the delay on this, finally got a chance to take a deeper look. For the maxplayers issue, how about this? It doesn't need anything in the command to run, right?

![image](https://github.com/seDirector/App/assets/49938263/03bbc01f-4a59-4ac0-9534-1e6bb6499997)

**Comment by asheroto** [comment]
@vanja-san did you mean `gamemodes_server.txt`? from here https://developer.valvesoftware.com/wiki/Maxplayers

And it's only for CS:GO right?

Revised, will only show for CS:GO. Look good?

![image](https://github.com/seDirector/App/assets/49938263/423c208a-e82f-4422-ba13-8fdb31d1c2f4)

**Reply by vanja-san** [reply]
yes👍

**Reply by asheroto** [reply]
okay cool, this will be implemented next release
